### PR TITLE
ROS Coach Node and Robocup Messages Node

### PR DESCRIPTION
**Pull request by hrinn**
Adds a coach node and a node containing standard messages passed between coach, player, and ssl_vision 
:rocket:
:sparkles:

### CODE REVIEW

File: src/robocup_msgs/package.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>robocup_msgs</name>
+  <version>1.0.0</version>
+  <description>Messages used by ssl_vision, coach, and player nodes</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
```
Comment (willle3456) [inline src/robocup_msgs/package.xml]: Remove boilerplate comments

File: src/coach/scripts/coach.py [hunk 2]
```diff
@@ -1,23 +1,36 @@
 #!/usr/bin/env python
 
 import rospy
+from robocup_msgs.msg import *
 
 def coach():
```
Comment (willle3456) [inline src/coach/scripts/coach.py:6]: Pascal case

File: src/coach/package.xml [hunk 3]
```diff
@@ -37,7 +37,6 @@
   <!--   <build_depend>roscpp</build_depend> -->
```
Comment (willle3456) [inline src/coach/package.xml]: Remove boilerplate comments

**Review by willle3456 — CHANGES_REQUESTED** [review]
Some really small changes before merging

File: src/coach/scripts/coach.py (hunk 2 above)
Comment (hrinn) [reply src/coach/scripts/coach.py:6]: So RoboCup_Msgs instead?

File: src/coach/package.xml (hunk 3 above)
Comment (hrinn) [reply src/coach/package.xml]: Done for all package.xml and CMakeLists.txt files

File: src/coach/scripts/coach.py (hunk 2 above)
Comment (willle3456) [reply src/coach/scripts/coach.py:6]: oh, I though coach was a class 😃 . It's ok

**Review by willle3456 — APPROVED** [review]

**Pull request merged**